_includes/index-docs.html

```diff
@@ -51,11 +51,18 @@ <h1 class="title">{{ page.title }}</h1>
         </div>
       </Transition>
       <div class="grid-wrapper">
-        <div v-for="card in cardHits"
-            v-html="card.innerHTML"
-            class="grid__item docs-card"
-            :class="Array.from(card.classList)">
+        {% raw %}
+        <div v-for="card in cardHits" :class="card.className">
+          <h4><a :href="card.url" :target="card.url.startsWith('http') ? '_blank' : ''" v-html="card.title"></a></h4>
+          <div class="description" v-html="card.summary"></div>
+          <div class="keywords" v-html="card.keywords"></div>
+          <div class="categories" v-html="card.categories"></div>
+          <div class="origin" v-if="card.origin" v-html="card.origin"></div>
+          <div class="content-highlights">
+            <p v-for="contentLine in card.content" v-html="contentLine"></p>
+          </div>
         </div>
```

Comment:
There's also a `index-guides.html` for older versions... can you have a look at that?

I suspect it'll be annoying...